utils/utils.js

```diff
@@ -24,6 +27,98 @@ function deslugifyCollectionPage(collectionPageName) {
   )
 }
 
+// retrieve the tree item
+async function getRootTree(repo, accessToken, branchRef='staging') {
+  try {
+    const headers = {
+      Authorization: `token ${accessToken}`,
+      Accept: 'application/json',
+    };
+    // Get the commits of the repo
+    const { data: commits } = await axios.get(`https://api.github.com/repos/${GITHUB_ORG_NAME}/${repo}/commits`, {
+      params: {
+        ref: branchRef,
+      },
+      headers,
+    });
+    // Get the tree sha of the latest commit
+    const { commit: { tree: { sha: treeSha } } } = commits[0];
+    const currentCommitSha = commits[0].sha;
+
+    const { data: { tree: gitTree } } = await axios.get(`https://api.github.com/repos/${GITHUB_ORG_NAME}/${repo}/git/trees/${treeSha}${isRecursive ? `?recursive=1` : ''}`, {
```

Comment:
`isRecursive` is undefined

Reply:
Thanks for pointing that out - I was playing around with recursive retrieval earlier and forgot to remove it. Resolved in https://github.com/isomerpages/isomercms-backend/pull/88/commits/04d90118a7fd4c1f31f397f1d7079b5307845d4c